neonvm-daemon/cmd/main.go

```diff
@@ -26,24 +32,73 @@ func main() {
 	defer logger.Sync() //nolint:errcheck // what are we gonna do, log something about it?
 
 	logger.Info("Starting neonvm-daemon", zap.String("addr", *addr))
+	cpuHotPlugController := &cpuscaling.CPUSysFsStateScaler{}
+	srv := cpuServer{
+		cpuOperationsMutex:  &sync.Mutex{},
+		cpuSystemWideScaler: cpuHotPlugController,
```

Comment:
I agree to rename cpuHotPlugController to something (or actually drop intermediate variable here at all), but I don't want to rename "cpuSystemWideScaler" to "cpuSysfsScaler", may be "cpuScaler"? 

because later there would be also something to ensure cgroup limits and my idea was that in the code it would something like:

```pseudocode
s.cpuScaler.EnsureOnlineCPUs(cpuN) // n is non fractional, may be could be EnsureCPUs
s.cgroupsScaler.SetCGroupLimit(cgroupsN) // cgroupsN is fractional and by design lower or equal to cpuN
```

Reply:
Yes, `cpuScaler` sounds good, I just wanted `SystemWide` out of the way, to not introduce new concepts.